Runtime/AvatarCreator/Scripts/UI/Elements/LogoutElement.cs

```diff
@@ -32,11 +34,15 @@ private async void Logout()
             AuthManager.Logout();
             try
             {
-                await AuthManager.LoginAsAnonymous();
+                await AuthManager.LoginAsAnonymous(cancellationTokenSource.Token);
                 LogOutSucceeded();
             }
             catch (Exception e)
```

Comment:
you could also use this 
```suggestion
            catch (Exception ex) when (ex is not OperationCanceledException)
```

Reply:
It's not OperationCanceledException for requests

Reply:
response exceptions are regular exceptions, not OperationCanceledExceptions
https://github.com/readyplayerme/rpm-unity-sdk-core/blob/main/Runtime/AvatarCreator/Scripts/Extensions/ResponseExtensions.cs